### PR TITLE
feat: routing and question page is added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
A question Page is added to this PR.
Routing paths are also added. 
To view the question page, a temporary link is created. Click on the link to go to the question page. As marked in the screenshot.
Then Link to the question page is temporary, it will be removed after this PR.
Screenshot (122)_LI

### CODE REVIEW

**Review by Nagashree01 — APPROVED** [review]
Code is clean and readable. Just a suggestion. You can make a separate file for routes as it is best practice. Read in an article. Ref: https://www.xenonstack.com/insights/reactjs-project-structure

Good to merge

**Comment by ShreyashMahajan** [comment]
> Code is clean and readable. Just a suggestion. You can make a separate file for routes as it is best practice. Read in an article. Ref: https://www.xenonstack.com/insights/reactjs-project-structure
> 
> Good to merge

thank you so much, i will apply that

File: src/components/navbar/navbar.jsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
     return (
         <div className="navbar">
             <div className="nav-container">
-                <h3 className="nav__heading"> Quizy </h3>
+                <NavLink to='/'><h3 className="nav__heading"> Quizy </h3></NavLink>
                 {/* <img className="nav--logo" alt="logo" /> */}
                 {true &&
                     <div className="nav--user-auth">
-                        <button className="btn btn--login">Login</button>
+                        <NavLink to='/login'> <button className="btn btn--login">Login</button> </NavLink>
```
Comment (SaurabhChawda) [inline src/components/navbar/navbar.jsx:13]: if you are not using style in NavLink, You can use Link.

File: src/pages/home/home.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { NavLink } from 'react-router-dom';
 import { CategoryCard } from '../../components/categoryCard/categoryCard';
 import { Navbar } from '../../components/navbar/navbar';
 import { Sidebar } from '../../components/sidebar/sidebar';
```
Comment (SaurabhChawda) [inline src/pages/home/home.jsx:4]: You can create an index.js file, and import all the components through index.js.

**Review by SaurabhChawda — APPROVED** [review]
UI looks good to me, You can Merge it.

File: src/pages/home/home.jsx (hunk 2 above)
Comment (ShreyashMahajan) [reply src/pages/home/home.jsx:4]: ok thank you

**Pull request merged**